api/endpoints/singlepoint_route.py

```diff
@@ -48,13 +40,25 @@ def get_singlepoint(request: SinglePointRequest) -> dict[str, Any]:
     """
     base_dir = Path("data")
     struct_path = base_dir / request.struct
-    logger.info("Request contents:", request)
+    logger.info("Request contents: %s", request)
```

Comment:
By the way, slightly different to this change, but this describes a bit about issues with f-strings etc. for logging, which isn't immediately obvious: https://docs.astral.sh/ruff/rules/logging-f-string/

This actual change is simpler since it's just setting the format.

(You can resolve this, was just for reference)